reference/apm/cloud/apm-settings.md

```diff
@@ -53,6 +53,10 @@ If a setting is not supported by {{ech}}, you will get an error message when you
 Some settings that could break your cluster if set incorrectly are blocklisted. The following settings are generally safe in cloud environments. For detailed information about APM settings, check the [APM documentation](/solutions/observability/apm/configure-apm-server.md).
 ::::
 
+### Version 9.1+ [ec_version_9_1]
```

Comment:
This config also applies to `8.19+` but I left it out based on @carsonip [comment](https://github.com/elastic/docs-content/pull/1269#discussion_r2075908722) in another PR. Let me know if I should add `8.19+`.

Reply:
@florent-leborgne @colleenmcginnis My initial plan was to backport this PR to 8.X branch for the 8.19 release (and change the versions from 9.1 to 8.19). But I just realized 8.19 is being released before 9.1. 

Should I create a separate PR for 8.X? Or do you have any other suggestions? Thanks

Reply:
Hey @isaacaflores2. Thanks for this PR. You would need a different PR anyways for 8.19 docs because:
- the content is likely in a different repository (https://github.com/elastic/observability-docs)
- 8.x docs, including 8.19, are still powered by the asciidoc-based system, while 9.0 docs and above like this PR are markdown-based.

I am happy to help if you need